src/System.Drawing.Common/src/System/Drawing/GdiplusNative.Unix.cs

```diff
@@ -32,7 +32,7 @@ private static IntPtr LoadNativeLibrary()
                 if (RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
```

Comment:
@jkotas, can this part (and the similar one in `LibcupsNative.cs`) be rewritten to extract the library name from `Interop.Libraries` constants defined in platform specific partial classes? This way we will not have if/else platform detection.

Reply:
It would require building Linux-specific and macOS-specific builds of the library. I think it is fine the way it is now.